index.js

```diff
@@ -59,14 +59,60 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+const h1 = document.querySelector('h1')
```

Comment:
Great! We do already have the "header" variable available for us to use, that we defined earlier in the file :)